app/views/godmin/resource/_button_actions.html.erb

```diff
@@ -1,3 +1,3 @@
-<% if policy(@resources).create? %>
+<% if policy(@resource_service.build_resource({})).new? %>
```

Comment:
Maybe this comment belongs in the other PR, but should we change `build_resource` to default the params to {} so we don't need to pass it in.